### PR TITLE
Add section on managing 18F resources to the team practices doc

### PR DESCRIPTION
**Pull request by konklone**
This attempts to channel our discussion in #33 into a set of guidelines for the team that hopefully reduce ambiguity over how we manage 18F resources and placement of repositories.

cc @janearc @benbalter @kaitlin

### CODE REVIEW

**Comment by afeld** [comment]
Well done :+1:

**Comment by seanherron** [comment]
Although this may just be advice, it does read like policy to me, and some of this touches on using government time/equipment for things which may have potential personal gain. That in mind, I would encourage having GSA Ethics take a look at this and ensure we're not saying something that could get a member of the team in trouble.

**Comment by konklone** [comment]
I understand that perspective, but I think it's going to be far easier to talk about around a specific issue. This guidance refers several times to teammates exercising professional judgment over whether something is of benefit to 18F or not. Ultimately, that's always how we're managing our open source work and boundaries -- and that professional judgment is of course subject to oversight and review. This guidance doesn't, IMO, suggest that that oversight or review is being eclipsed or marginalized.

What about an explicit line that says something like:

> Any teammate who is not certain over whether some work would meet the criteria of primarily benefiting 18F should reach out to the broader 18F team or the GSA Ethics team for clarity.

**Comment by seanherron** [comment]
Is "primarily benefiting 18F" the sole criteria Ethics would use when determining if an individual was acting in good faith or not? If so, I think that would address my concerns, but I'm not sure that is the benchmark behind which a judgement would be made.

**Comment by gbinal** [comment]
Something explicit like that would help, @konklone.  Otherwise, I think this edit is useful and I don't think the general guidance needs signoff.

**Comment by konklone** [comment]
Ah, rereading your comments, you pointed more to conflicts of interest. I think you're right that people should be aware that this is the dynamic at work that Ethics would be concerned about. 

Second draft:

> Any teammate who is not certain whether some work would sufficiently and primarily benefit 18F, or has a concern about a conflict of interest, should reach out to the broader 18F team or the GSA Ethics team for clarity.

**Comment by seanherron** [comment]
@konklone that's what I'm getting at - what other "ands" apply here? Are there additional criteria or situations where someone should talk to the GSA Ethics team? We don't know unless we talk to them.

File: practice.md [hunk 1]
```diff
@@ -74,6 +74,25 @@ When an 18F project has a non-standard license status (e.g. it's a fork of a pre
 
 All 18F team members are expected to make new source code repositories public from the time of creation.
 
+### Managing 18F resources
+
+18F intends to produce great software for the American people. That means not just rushing through projects to get them working as fast as possible, but managing technical debt with an eye towards usability and reusability.
```
Comment (janearc) [inline practice.md]: I think "technical debt" here should be at the least hyperlinked or even defined in a very short parenthetical abstract.

Comment (konklone) [inline practice.md]: Done in 3913f61.

**Comment by kaitlin** [comment]
I am 100% ok with the language as written. I think we would mostly need to be concerned about monetary benefit to an individual w/r/t ethics. As it's written, I think it sets expectations that we maintain and improve our open source projects in the context of "shipping" something. Removing technical debt is "shipping" something, as is adding any features we might use in the development of a product. 

I also think it's ok to set the expectation that feature requests that are not germane to our products are great candidates for the open source community to tackle, but that in typical agile fashion, we probably won't address them until there's a need from our end.

**Comment by cmc333333** [comment]
:+1:

**Comment by gbinal** [comment]
:+1:

**Pull request merged**